### PR TITLE
✨ Add home page

### PR DESCRIPTION
**Pull request by SilviaBarcellini**
- build and reboot home page
- import components
- add links to other pages
- style home page
- make home page responsive

### CODE REVIEW

**Review by mrana17 — APPROVED** [review]
Lgtm 🚀

**Review by DNaunin — APPROVED** [review]
Great Job! 🚀

File: pages/Form.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import Footer from "../components/footer/footer";
+import Form from "../components/forms/form";
+
+export default function MainCard() {
+  return (
+    <div>
+      <div>
+        <Header logo="bakend" imageSrc="/donuts.svg" />
+      </div>
+      <div className={styles.container}>
```
Comment (phlgr) [inline pages/Form.tsx:12]: Try removing the `min-width: 100vw` here to get rid of the vertical scroll bar

File: package.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
     "build-server": "tsc --project tsconfig.server.json"
   },
   "dependencies": {
+    "mongodb": "^3.6.5",
```
Comment (phlgr) [inline package.json:22]: 👍

File: pages/index.tsx [hunk 3]
```diff
@@ -16,23 +15,25 @@ export default function Home() {
...
-            <Button primary label="Primary" />
+            {/* <Button primary label="Primary" />
             <Button primary={false} label="Secondary" />
-            <OpenNewPage />
-            <div>
-              <a href="/mainCard">
-                press here to navigate to the Main Card Page (temporary link)
-              </a>
-            </div>
+            <OpenNewPage /> */}
```
Comment (phlgr) [inline pages/index.tsx:21]: Remove this comment

File: pages/index.tsx [hunk 4]
```diff
@@ -16,23 +15,25 @@ export default function Home() {
...
+            <a href="/Form">
+              <Btn label="Today I am out of ..." />
+            </a>
+            <a href="/mainCard">
+              <Btn label="Discover New Ingredients!" />
+            </a>
           </div>
+          {/* <div>
+            <Form />
+          </div> */}
```
Comment (phlgr) [inline pages/index.tsx:36]: Is this comment needed here?

File: styles/Form.module.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
+.container {
+  min-height: 100vh;
+  min-width: 100vw;
```
Comment (phlgr) [inline styles/Form.module.css:3]: This is causing vertical scroll bars

File: styles/Home.module.css [hunk 6]
```diff
@@ -7,11 +7,17 @@
...
 
-.main {
+/* .main {
   padding: 5rem 0;
   flex: 1;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
+} */
```
Comment (phlgr) [inline styles/Home.module.css:17]: Remove commented code

File: pages/index.tsx [hunk 7]
```diff
@@ -16,23 +15,25 @@ export default function Home() {
...
           </div>
-          <div>
-            <Form />
+          <div className={styles.btn}>
+            <a href="/Form">
+              <Btn label="Today I am out of ..." />
+            </a>
+            <a href="/mainCard">
+              <Btn label="Discover New Ingredients!" />
+            </a>
```
Comment (phlgr) [inline pages/index.tsx:32]: These should be `Link` elements. Take a look here:
https://nextjs.org/docs/api-reference/next/link

File: styles/Form.module.css [hunk 8]
```diff
@@ -0,0 +1,21 @@
+.container {
+  min-height: 100vh;
```
Comment (phlgr) [inline styles/Form.module.css:2]: Why is this height here?

**Review by phlgr — CHANGES_REQUESTED** [review]
You should fix the Navbar to the bottom of the page, so it is easier to navigate, and you don't have to scroll all the way down first.
Gj 🚀

**Pull request merged**